xds/internal/server/rds_handler_test.go

```diff
@@ -18,25 +18,27 @@
 
 package server
 
+/*
```

Comment:
I'm guessing this test is not ready to be looked at.

Reply:
Added.